tests/cypress/e2e/features/issue_4662_Count_Annotations.js

```diff
@@ -0,0 +1,75 @@
+import { taskName, labelName } from '../../support/const';
+
+context('Count total annotation, issues and labels', () => {
+    const issueId = '4662';
+    const createRectangleShape2Points = [{
+        points: 'By 2 Points',
+        type: 'Shape',
+        labelName,
+        firstX: 250,
+        firstY: 350,
+        secondX: 350,
+        secondY: 450,
+    },
+    {
+        points: 'By 2 Points',
+        type: 'Shape',
+        labelName,
+        firstX: 500,
+        firstY: 100,
+        secondX: 700,
+        secondY: 200,
+    },
+    {
+        points: 'By 2 Points',
+        type: 'Shape',
+        labelName,
+        firstX: 400,
+        firstY: 300,
+        secondX: 450,
+        secondY: 400,
+    },
+    ];
```

Comment:
I believe we can just draw the same object several times, so, we do not need to defined it with different positions.